pkg/tfbridge/provider.go

```diff
@@ -66,6 +69,16 @@ type Provider struct {
 	memStats        memStatCollector
 }
 
+// ProviderMixin defines an interface which must be implemented by providers
```

Comment:
Self-reference wasn't renamed in the doc comment.

Reply:
Fixed